roles/docker/templates/crio.conf.j2

```diff
@@ -103,12 +103,17 @@ cgroup_manager = "systemd"
 # hooks_dir_path is the oci hooks directory for automatically executed hooks
 hooks_dir_path = "/usr/share/containers/oci/hooks.d"
 
+# default_mounts is the mounts list to be mounted for the container when created
+default_mounts = [
+	"/run/secrets",
```

Comment:
This is wrong, should be:
```
/usr/share/rhel/secrets:/run/secrets
```

Reply:
This will allow subscription manager to work inside a container

Reply:
thanks, I've just updated it